File: c#/leftbeehind.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Numerics;
using System.Threading;

namespace Kattis
{
    public class Program
    {
        public static void Main()
        {
            while (true) {
               var x = Console.ReadLine().Split().Select(y => int.Parse(y)).ToArray();

                if (x[0] == 0 && x[1] == 0)
                    break;
                else if (x[0] + x[1] == 13)
                    Console.WriteLine("Never speak again.");
                else if (x[0] > x[1])
                    Console.WriteLine("To the convention.");
                else if (x[0] < x[1])
                    Console.WriteLine("Left beehind.");
                else
                    Console.WriteLine("Undecided.");
            }
        }
    }
}

File: python3/leftbeehind.py
while True:
    x = input().split()
    x[0] = int(x[0])
    x[1] = int(x[1])
    if x[0] == 0 and x[1] == 0:
        break
    if x[0] + x[1] == 13:
        print("Never speak again.")
    elif x[0] > x[1]:
        print("To the convention.")
    elif x[0] < x[1]:
        print("Left beehind.")
    else:
        print("Undecided.")
